package.json

```diff
@@ -29,23 +29,23 @@
     "class-transformer": "^0.3.1",
```

Comment:
I see you haven't updated the apollo-server version here but it is updated in https://github.com/UserOfficeProject/user-office-scheduler-backend/pull/11/files. Is there any reason not to keep them aligned?

Reply:
There was a dependency version mismatch and the [TS types were in conflict](https://github.com/UserOfficeProject/user-office-gateway/pull/9/checks?sha=99a949b07190bc3eb16f3eb04f740426f8f6da52) so I skipped it for now